MediaAdminBundle/Resources/config/form.yml

```diff
@@ -14,11 +14,26 @@ parameters:
                 icon: false
         use_callback_tinymce_init: true
 
+    open_orchestra_media_admin.allowed_mime_type:
```

Comment:
I would rather put this parameters as a configuration of the bundle, this way integrator could add new mime types